games/templates/play_game_as_player.html

```diff
@@ -123,6 +123,42 @@ <h5 class="modal-title">Do you want to buy <span id="cardName"></span></h5>
     </div>
 </div>
 
+<div id="draw-modal" class="modal" tabindex="-1" role="dialog">
```

Comment:
We can use these modals to update our integration tests to test what is in the draw pile (Stuff like bureaucrat putting stuff on the top of a deck) and gaining cards to discard pile. 

Note: Draw pile is sorted by cost then ABC order. So the user will not be able to see what is next. Discard pile is "in order" but matters much less.